<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BenchmarkWindow</class>
 <widget class="QMainWindow" name="BenchmarkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>782</width>
      <height>539</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>751</width>
          <height>251</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>260</y>
          <width>758</width>
          <height>254</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="Correction">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="Choice_C">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="Choice_A">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="Choice_B">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Menu</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Next">
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="Choice_D">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="Retry">
           <property name="text">
            <string>Try Again</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="Score">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>321</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
